src/core/tests/library/RuntimeCompositor.py

```diff
@@ -45,6 +47,14 @@ def __init__(self, argv=Constants.DEFAULT_UNSPECIFIED_VALUE, legacy_mode=False,
         self.argv = argv if argv != Constants.DEFAULT_UNSPECIFIED_VALUE else ArgumentComposer().get_composed_arguments()
         self.vm_cloud_type = vm_cloud_type
         Constants.Paths.SYSTEMD_ROOT = os.getcwd() # mocking to pass a basic systemd check in Windows
+        self.is_github_runner = os.getenv('RUNNER_TEMP', None) is not None
```

Comment:
Github runners set this. Ben found this in a random issues/PR trying to fix things.